### DISCUSSION TITLE
Musics fonts do not load correctly on Windows 10

### DISCUSSION THREAD

**Discussion by dfcompose** [cac-t-u-s/om-sharp#358] [Reports]
Bravura does not load correctly on Windows 10, using large Xs for noteheads and no clefs.

![296086984-faefec15-2b92-4739-bdec-7aad70408d20](https://github.com/cac-t-u-s/om-sharp/assets/133391238/f6778f38-1ecf-495f-81fe-b6e5d340ddd2)

In addition, when installed there is no font file generated in the "fonts" folder in the om-sharp directory, only a "bravura_metadata" json file.

![image](https://github.com/cac-t-u-s/om-sharp/assets/133391238/4cff4397-e793-4a82-a3f7-8a495fbacbac)

**Comment by j-bresson** [comment]
@afarra6 sorry for the delayed reply. 

This is not good: the installer shoudl have created a font file here. You can add it manually though: just download the files from the sources here https://github.com/cac-t-u-s/om-sharp/tree/master/resources/fonts, or directly on the [Bravura repository](https://github.com/steinbergmedia/bravura)

**Comment by dfcompose** [comment]
Downloaded and reinstalled the fonts and placed the files in the fonts folder as well. Unfortunately the same issue is still happening. OM# can find the font (I think?) as I can set the listener text to be Bravura/Bravura Text:

![image](https://github.com/cac-t-u-s/om-sharp/assets/133391238/2895d85a-e2e3-4198-b3f3-d6db99e58644)


But the notation isn't appearing correctly. Thanks for your help as well!

**Reply by j-bresson** [reply]
Have you tried to double-click the Bravura.ttf file in order to setup the font?

**Reply by dfcompose** [reply]
Yes, I tried the .ttf first and then the .otf, opening OM# between font installs, and the fonts still did not appear in OM#. Is there somewhere that I can check to see if OM# is looking for Bravura in the right place?

**Reply by j-bresson** [reply]
not really, I think OM# is just using font that are available; it doesn't look at a particular place as far as I can tell... 🤔

**Reply by dfcompose** [reply]
Hmmm if there's anything I can check on my end let me know! For what it's worth Bravura works fine with Dorico on my computer.
